pkg/aws/aws.go

```diff
@@ -122,16 +106,23 @@ func NewClientFromFileCredentials(dir string, region string) (*SdkClient, error)
 	}
 	accessKeyID := strings.TrimRight(string(accessKeyBytes), "\n")
 	secretKeyID := strings.TrimRight(string(secretKeyBytes), "\n")
+	// XXX: Does an empty session token work?
 	return NewClient(accessKeyID, secretKeyID, "", region)
 }
 
+// FIXME: This seems quite useless
+// GetAWSCredentials gets the AWS credentials
+func (c *SdkClient) GetAWSCredentials() awsv2.Credentials {
+	return c.Credentials
+}
+
```

Comment:
It's actually needed after all, because we only have the interface available - but to access the field it would need the struct, so the indirection is needed.